src/index.ts

```diff
@@ -0,0 +1,10 @@
+export class Challenge {
```

Comment:
🙈We use Allman style brackets (https://en.wikipedia.org/wiki/Indentation_style#Allman_style).
Put them on a new line